build.gradle

```diff
@@ -47,6 +47,7 @@ task downloadIntelDeflater(type: Download) {
     //IntelDeflater is not packages with the rest of HTSJDK so we download it by hand
     src 'https://github.com/samtools/htsjdk/raw/2.1.1/lib/jni/libIntelDeflater.so'
     dest 'build/'
+    overwrite false
```

Comment:
it's a tradeoff between downloading a fresh copy every single time (which extends the installDist time by a reasonable amount) or to only download a fresh copy after a gradle clean (which seems reasonablish to me...)  I don't know if it tracks changes to the src url.  It's possible that it will always redownload if the source url changes which is the ideal situtation.  I'll experiment and see.